RCS file: /pack/cvsroots/wxwindows/wxWindows/include/wx/filesys.h,v
retrieving revision 1.21
diff -u -r1.21 filesys.h
--- filesys.h	2003/07/08 19:52:20	1.21
+++ filesys.h	2003/07/17 14:52:51
@@ -161,7 +161,7 @@
 {
 public:
     wxFileSystem() : wxObject() {m_Path = m_LastName = wxEmptyString; m_FindFileHandler = NULL;}
-    ~wxFileSystem() { WX_CLEAR_LIST(wxList, m_Handlers); }
+    ~wxFileSystem() {}
 
     // sets the current location. Every call to OpenFile is
     // relative to this location.


 	  	 
